lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -124,19 +147,30 @@ window.runLighthouseForConnection = function(connection, url, options, requested
   lighthouseIsRunning = true;
   updateBadgeUI(url);
 
-  // Run Lighthouse.
-  return Runner.run(connection, runOptions)
-    .then(result => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      filterOutArtifacts(result);
-      return result;
-    })
-    .catch(err => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      throw err;
-    });
+  return enableOtherChromeExtensions(false)
+      .then(_ => Runner.run(connection, runOptions)) // Run Lighthouse.
+      .then(result => {
+        lighthouseIsRunning = false;
+        updateBadgeUI();
+        filterOutArtifacts(result);
+
+        // Deliver results even if enabling extensions fails.
+        enableOtherChromeExtensions(true).catch(err => {
+          log.warn('Chrome', `Could not disable some extensions. ${err.message}`);
```

Comment:
Done. For the other, I don't think it's a good UX if LH fails because an extension fails to enable. That was my intention at least. I'm not sure how enabling _could_ fail, but users can still get results if they do.